docs/source/notebooks/climex.ipynb

```diff
@@ -1866,93 +1868,6 @@
    "cell_type": "code",
    "execution_count": 6,
    "metadata": {},
-   "outputs": [],
-   "source": [
-    "# This will return instantaneously, because xclim is using lazy evaluation whenever possible.\n",
-    "sdii = xclim.atmos.daily_pr_intensity(\n",
-    "    ds.pr.isel(realization=[4, 5], rlat=slice(100, 200), rlon=slice(100, 200)).sel(\n",
```

Comment:
Oh this PR is about formatting, normal this section of code is removed?

Reply:
That's not normal, no. It may have been a strange merge of master ? Will look into this right away.